docs/topics/con_validating-guides.adoc

```diff
@@ -39,3 +39,26 @@ To re-enable validation of the book, remove the `.ci-ignore` file, for example:
 $ rm docs/wf-swarm-runtime/.ci-ignore
 ----
 
+== Checking for Unused Topics
```

Comment:
Zach, can you please move this into a separate module? This is not related to validation IMO.